index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="body-wrapper">
+      <!-- nav section -->
+      <div class="mobile-nav">
+        <h1>Welcome</h1>
+        <img src="./assets/main-images/bars.png" alt="" />
+      </div>
+      <div class="hero-wrapper">
+        <nav class="desktop-nav">
+          <ul>
+            <li>
+              <a href="/">Porfolio</a>
+            </li>
+            <li>
+              <a href="/">About</a>
+            </li>
+            <li>
+              <a href="/">Contact</a>
+            </li>
+          </ul>
+        </nav>
+        <!-- hero section -->
+        <section class="hero-section">
+          <div class="container">
+            <div class="content">
+              <h1>
+                Hey there. I’m Alex. <br />
+                <span>I’m a software developer</span>
+              </h1>
+              <p>
+                I can help you build a product , feature or website Look through
+                some of my work and experience! If you like what you see and
+                have a project you need coded, don’t hestiate to contact me.
+              </p>
+            </div>
+            <div class="socials-wrapper">
+              <div class="socials-container">
+                <a href="/">
+                  <img src="./assets/main-images/github.png" alt="" />
+                </a>
+              </div>
+              <div class="socials-container">
+                <a href="/">
+                  <img src="./assets/main-images/linkedin.png" alt="" />
+                </a>
+              </div>
+              <div class="socials-container">
+                <a href="/">
+                  <img src="./assets/main-images/hand.png" alt="" />
+                </a>
+              </div>
+              <div class="socials-container">
+                <a href="/">
+                  <img src="./assets/main-images/twitter.png" alt="" />
+                </a>
+              </div>
+              <div class="socials-container">
+                <a href="/">
+                  <img src="./assets/main-images/M.png" alt="" />
+                </a>
+              </div>
+            </div>
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. Kindly ensure to apply that ☺️ .
- [ ] In addition, The required `alt` attribute specifies an alternate text for an image if the image cannot be displayed. The alt attribute provides alternative information for an image if a user for some reason cannot view it (because of a slow connection, an error in the src attribute, or if the user uses a screen reader) ☺️ . I suggest that you provide a descriptive `alt text` 👍 .

Reply:
@AlphaNtihinduka Thanks a lot for your time and comment used to review this work. I really appreciate please.